mapDownload.properties

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
This is a new file that tells the game where to get triplea_maps.xml

To test this feature you'll need to comment out the github line and uncomment the local or source forge URLs.